examples/dataproc-spanner/jdbc/src/main/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
+Main-Class: org.tonyz.com.Main
```

Comment:
You could use com.google.cloud.pso as package name

Reply:
Of course, this has been updated.